src/test/unit/specific_feature_tests/image-resize-handle.js

```diff
@@ -127,3 +129,17 @@ test("resize handle with no image is hidden on `mousemove`", function () {
         }
     });
 });
+
+test("resize handle detached before adding history", function () {
+    expect(1);
+    var wymeditor = jQuery.wymeditors(0);
+    var spy = sinon.spy(wymeditor.ih, '_detachResizeHandle');
+
+    wymeditor.undoRedo.reset();
+    wymeditor.html('<p><img src="' + IMG_SRC + '" /></p>');
+    wymeditor.$body().find('img')
+        .mousemove();
+    wymeditor.undoRedo._add();
+
+    strictEqual(spy.calledOnce, true);
```

Comment:
You're testing that _detachResizeHandle is called once, which doesn't seem the same as testing that it's not added to the undo/redo stack to me.